src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -354,21 +354,28 @@ public function filter_add_body_classes( $classes ) {
 	 * @param string $file      Complete path to include the PHP File
 	 * @param array  $name      Template name
 	 * @param object $template  Instance of the Tribe__Template
-	 * 
+	 *
 	 * @return string
 	 */
 	public function filter_template_file( $file, $name, $template ) {
-		
-		if ( 
+		if (
 			! is_singular( TEC::POSTTYPE )
 			&& 'single-event' !== tribe_context()->get( 'view' )
 		) {
 			return $file;
 		}
-		
-		$override = V1_Event_Templates::getTemplateHierarchy( 'default-template' );
-		
-		return ! empty( $override ) ? $override : $file;
+
+		$theme_file = locate_template( 'tribe-events/default-template.php', false, false );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "FALSE" but found "false"
* :no_entry_sign: TRUE, FALSE and NULL must be uppercase; expected "FALSE" but found "false"
* :warning: Line exceeds 80 characters; contains 91 characters

<sub>hash: 8634e72445570c01b522e0aa73644332</sub>